deepchem/feat/tests/test_protein_tokenizer.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import numpy as np
+
+from deepchem.feat import ProteinTokenizer
+
+
+class TestProteinTokenizer(unittest.TestCase):
```

Comment:
Can you add a few FASTA example files and check that the featurizer can featurize lines from these files?

Reply:
Certainly possible!

I'd have to do some file processing in the test file so to accomplish that, since `ProteinTokenizer` only takes arrays of strings.

Also, it would be pretty hard to check that the tokenization was accurate, since its quite difficult to tokenize a large string by hand.